### DISCUSSION TITLE
Find out current version

### DISCUSSION THREAD

**Discussion by ghostdevv** [Budibase/budibase#1816] [Help] [answered]
I can't figure out where to see the current version of my installation

**Comment by shogunpurple** [comment, marked as answer]
@ghostdevv on the latest version you can go to `Updates` on the admin panel, and you will see which version of budibase you are running.

![Screenshot 2021-06-23 at 22 47 18](https://user-images.githubusercontent.com/11256663/123172372-f82e2280-d474-11eb-82a3-d48146aa5b64.png)

**Reply by oswaldB** [reply]
I don't have this information shown on my update panel?

**Reply by shogunpurple** [reply]
@oswaldB you might have to update with the CLI first budi —hosting update to get this feature if you are on an older version.

**Comment by oswaldB** [comment]
Hello,

![image](
https://user-images.githubusercontent.com/3023526/123386786-f062a100-d58e-11eb-94d7-3050698f81b7.png
)

I cannot update any idea why?

On Fri, 25 Jun 2021 at 07:32, Martin McKeaveney ***@***.***>
wrote:

> @oswaldB <https://github.com/oswaldB> you might have to update with the
> CLI first budi —hosting update to get this feature if you are on an older
> version.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-918819>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJQT2URDDS3WNP7UQBDTUQPJHANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by shogunpurple** [reply]
What does the `docker-error.log` say?

**Comment by oswaldB** [comment]
I don't know where it is...

On Fri, 25 Jun 2021 at 08:27, Martin McKeaveney ***@***.***>
wrote:

> What does the docker-error.log say?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919016>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJRNZAHFRSRX2HBRCYLTUQVWDANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by shogunpurple** [reply]
`cat docker-error.log` in the same directory you ran `budi hosting --update`

**Comment by oswaldB** [comment]
[image: image.png]


On Fri, 25 Jun 2021 at 08:31, Martin McKeaveney ***@***.***>
wrote:

> cat docker-error.log in the same directory you ran budi hosting --update
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919034>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJUHDJNKBMJMTRFG6ALTUQWGRANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by shogunpurple** [reply]
That's not a valid image - I can't see it

**Comment by oswaldB** [comment]
it says no such file or directory.

On Fri, 25 Jun 2021 at 08:35, Martin McKeaveney ***@***.***>
wrote:

> That's not a valid image - I can't see it
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919047>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJWDZPRHLKR5N2FNNX3TUQWUDANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by shogunpurple** [reply]
Please paste the result of an `ls` in the `bruxelles` directory

**Comment by oswaldB** [comment]
docker-compose.yaml  envoy.yaml  logs

the logs folder is empty.

PS: in the meantime, I setup a new instance and will upload the exported
apps.

On Fri, 25 Jun 2021 at 08:39, Martin McKeaveney ***@***.***>
wrote:

> Please paste the result of an ls in the bruxelles directory
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919058>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJUY26CSSRFOVGNZ35TTUQXCJANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by shogunpurple** [reply]
One thing you could try as well is just a `budi hosting --stop` and a `docker-compose up` which should show you the error messages in the console

**Comment by oswaldB** [comment]
It provides no error at all ;(

On Fri, 25 Jun 2021 at 08:45, Martin McKeaveney ***@***.***>
wrote:

> One thing you could try as well is just a budi hosting --stop and a docker-compose
> up which should show you the error messages in the console
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919073>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJVPGFZ7YVTYVY3S5W3TUQXY3ANCNFSM47GUE6XA>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Comment by oswaldB** [comment]
I set up a new instance and now it works perfectly I have access ot the
latest version.

Thanks for your help

On Fri, 25 Jun 2021 at 08:51, oswald bernard ***@***.***>
wrote:

> It provides no error at all ;(
>
> On Fri, 25 Jun 2021 at 08:45, Martin McKeaveney ***@***.***>
> wrote:
>
>> One thing you could try as well is just a budi hosting --stop and a docker-compose
>> up which should show you the error messages in the console
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Budibase/budibase/discussions/1816#discussioncomment-919073>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAXCFJVPGFZ7YVTYVY3S5W3TUQXY3ANCNFSM47GUE6XA>
>> .
>>
>
>
> --
> Oswald BERNARD
> 00 44 7861004200
>


-- 
Oswald BERNARD
00 44 7861004200
